### PR TITLE
Website: Bold Application Text and Fix Application Links

### PR DESCRIPTION
**Pull request by andrew032011**
### Summary 
~~We received leads feedback to show the “We are no longer accepting applicants” text everywhere where the apply button typically would be, so it doesn’t feel like a bug when applicants navigate back to the page and don’t see the apply button anymore.~~ We're going to redo this design to instead include a banner. Handled in a separate PR.

Also, fix application links on the Apply timeline.

### Notion/Figma Link 
https://www.notion.so/Website-Bold-Application-Text-and-Fix-Application-Links-1440ad723ce180e1879dc8bce2e069fe?pvs=4

### Test Plan 
The two "Google Form Application" links lead to the application now.

### CODE REVIEW

**Review by JasonMun7 — APPROVED** [review]
Hey Andrew! The changes look good to me! I just left some very small opinionated comments! However, feel free to ignore as I'm sure Leads will also have other ideas!

File: new-dti-website/src/app/apply/page.tsx [hunk 1]
```diff
@@ -39,7 +39,7 @@ const ApplyHero = () => (
             Apply now
           </Link>
         ) : (
-          <p className="md:text-lg xs:text-sm">
+          <p className="md:text-lg xs:text-sm font-bold">
```
[the remark this answers is not in the document]
Comment (andrew032011) [reply new-dti-website/src/app/apply/page.tsx:42]: I think that could work! Curious to hear @Bookie0's thoughts though.

[the remark this answers is not in the document]
Comment (Bookie0) [reply new-dti-website/src/app/apply/page.tsx:42]: Hi Jason! Thanks for your comment!

I replied in the Slack [here](https://cornelldti.slack.com/archives/C036VK3BRDE/p1732131893572779).

TL;DR, we could add a banner at the top of the hero section. Idk if we do need the icon tho cuz it feels a bit aggressive imo? It's like a "Do not Enter" or "do not Disturb" icon which feels a bit bold compared to the message we wanna say

**Comment by andrew032011** [comment]
Since the PR scope has changed (we're going to be adding a banner here): https://cornelldti.slack.com/archives/C036VK3BRDE/p1732131893572779

I'll merge just the link fixes in with this PR and have the banner be a separate change (@Bookie0 is going to sketch up a design).
